content/insecure_linux.rst

```diff
@@ -0,0 +1,63 @@
+################################################
+Fast But Insecure Linux Setup - Adding Grub Menu
+################################################
+:date: 2019-10-19
+:tags: linux
+:category: Computer
+:slug: fast-but-insecure-linux-setup-adding-grub-menu
+:author: John Nduli
+:status: published
+
+I got the idea of setting this up from this `article on making linux
+fast again
+<https://linuxreviews.org/HOWTO_make_Linux_run_blazing_fast_(again)_on_Intel_CPUs>`_.
+The problem was that I did not want to set up these flags on my default
+grub menu entry. I also did not understand what risks I'd be exposing
+myself to, but I could get their names by running:
+
+.. code-block:: bash
+
+    ls /sys/devices/system/cpu/vulnerabilities 
+
+    # which gave me
+    l1tf  mds  meltdown  spec_store_bypass  spectre_v1  spectre_v2
+
+The quick fix for the former problem was to add another menu entry in
+grub that had the `mitigations=off` flag. The easiest way I found for
+this was to use `grub-customizer
+<https://launchpad.net/grub-customizer>`_. I just copied the contents of
+the default grub setup to another profile, added the extra flag and
+named this to `Insecure ArchLinux`. Here's the configuration I used for
+this menu entry (notice the `mitigations=off` added to the third last
+line):
```

Comment:
Why install `grub-customizer` when you have steps for updating a copy of the default grub setup